### PR TITLE
[BD-14] feat: Update library listing page according to design

### PR DESCRIPTION
**Pull request by ruzniaievdm**
### Description:

- add breadcrumbs according to design. Add state for breadcrumb in the redux
- rename Title according to design
- restyle New library button
- display libraries with using selectable box as in the design
- display link from the Figma in the footer
- Pagination. Add pagination when there are more than 1 page is present. Add env variables for Library Listing page and Library Block Listing page count per page(50)
- add empty page if not exists libraries

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ruzniaievdm!


When this pull request is ready, tag your edX technical lead.

**Comment by ruzniaievdm** [comment]
@connorhaugh Please take a look!

File: .env [hunk 1]
```diff
@@ -18,3 +18,4 @@ USER_INFO_COOKIE_NAME=null
 SECURE_ORIGIN_XBLOCK_BOOTSTRAP_HTML_URL=null
 BLOCKSTORE_COLLECTION_UUID=null
 LOGO_TRADEMARK_URL=null
+LIBRARY_LISTING_PAGINATION_PAGE_SIZE=null
```
Comment (connorhaugh) [inline .env:21]: Presumably 2U will have to set this value for our Prod. What is the spec to set it at again? 40?

File: src/library-authoring/list-libraries/LibraryListPage.jsx [hunk 2]
```diff
@@ -207,80 +190,8 @@ export class LibraryListPage extends React.Component {
                     buttonLabels={paginationOptions.buttonLabels}
                     onPageSelect={this.handlePageChange}
                   />
-                )
-                : null}
+                )}
             </article>
-            <aside className="content-supplementary">
```
Comment (connorhaugh) [inline src/library-authoring/list-libraries/LibraryListPage.jsx:213]: this has been replaced by the AboutLibraraiesHyperLink, right?

File: src/vendor/studio.scss [hunk 3]
```diff
@@ -1130,15 +1129,25 @@ $bi-app-right           : right;
 // edx-platform/cms/static/sass/_base.scss
 html {
   font-size: 62.5%;
-  height: 102%; // force scrollbar to prevent jump when scroll appears, cannot use overflow because it breaks drag
+  height: 100%; // force scrollbar to prevent jump when scroll appears, cannot use overflow because it breaks drag
```
Comment (connorhaugh) [inline src/vendor/studio.scss:1132]: This is the default value I believe. Is there a reason to set it here? Also the comment no longer applies.

File: .env (hunk 1 above)
Comment (ruzniaievdm) [reply .env:21]: I think we decided that this value is 50

File: src/library-authoring/list-libraries/LibraryListPage.jsx (hunk 2 above)
Comment (ruzniaievdm) [reply src/library-authoring/list-libraries/LibraryListPage.jsx:213]: It isn't here, `AboutLibrariesHyperlink` is inserted at the root of `index.jsx`

File: src/vendor/studio.scss (hunk 3 above)
Comment (ruzniaievdm) [reply src/vendor/studio.scss:1132]: You are right comment isn't actual. `height: 100%` will be included in the next our pr's  regarding the task - `refactoring of remove global font-size from the html tag`. Comment also will be removed.

In this case here it's necessary for the correct display of the footer hyperlink.

Comment (connorhaugh) [reply src/vendor/studio.scss:1132]: Ah great, thank you.

File: src/library-authoring/list-libraries/LibraryListPage.jsx (hunk 2 above)
Comment (connorhaugh) [reply src/library-authoring/list-libraries/LibraryListPage.jsx:213]: Yes, so now we don't need this content here. I agree.

**Comment by connorhaugh** [comment]
Just clean up that one lint error in the CI and this looks good to go.

**Comment by ruzniaievdm** [comment]
@connorhaugh I fixed lint error, please can you restart the workflows?

**Review by connorhaugh — APPROVED** [review]
Looks good to me now. Feel free to merge, unless you do not have permission, and I need to do so?

**Comment by ruzniaievdm** [comment]
@connorhaugh you feel free to merge, there aren't blockers from our side

**Comment by openedx-webhooks** [comment]
@ruzniaievdm 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-library-authoring&entry.867055334=https://github.com/openedx/frontend-app-library-authoring/pull/58&entry.1484655318=https://github.com/ruzniaievdm&entry.752974735=2022-08-17+11:26&entry.1917517419=2022-08-24+14:43&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**